autode/utils.py

```diff
@@ -134,16 +135,25 @@ def wrapped_function(*args, **kwargs):
     return func_decorator
 
 
-def work_in_tmp_dir(filenames_to_copy, kept_file_exts, use_ll_tmp=False):
+def work_in_tmp_dir(filenames_to_copy: Optional[List[str]] = None,
+                    kept_file_exts:    Optional[List[str]] = None,
+                    use_ll_tmp:        bool = False):
     """Execute a function in a temporary directory.
 
     Arguments:
         filenames_to_copy (list(str)): Filenames to copy to the temp dir
 
         kept_file_exts (list(str): Filename extensions to copy back from
                        the temp dir
+
+        use_ll_tmp (bool): If true then use autode.config.Config.ll_tmp_dir
     """
     from autode.config import Config
+    if filenames_to_copy is None:
```

Comment:
Can't `filenames_to_copy` and `kept_file_exts` be set to `[]` by default?

Reply:
Mutable default arguments are bad, apparently https://docs.python-guide.org/writing/gotchas/

Reply:
Ah yes, actually I forgot that having `[]` as the default value is classic bad practice :-1: